### PR TITLE
Update NuGet references automatically once a week

### PR DESCRIPTION
**Pull request by sebastienros**
This will create a single PR weekly with all the minor version updates for the referenced packages.
If we think that some of them should be updated less often, or with different rules (no patch version for instance), or not updated automatically, then we can add more groups.

These PRs also reference the release notes from each package update if available, which helps us understand what we are getting.

### CODE REVIEW

**Review by Piedone — CHANGES_REQUESTED** [review]
This is great, but we'll need to keep the src/docs/resources/libraries/README.md file up-to-date too. Doing that manually while the packages auto-update automatically wouldn't be fun.

I'm OK with just removing the version columns there; I was never a big fan of that page and its maintenance burden anyway. Or perhaps we can remove it altogether, since you can generate such an overview anyway (https://github.com/OrchardCMS/OrchardCore/issues/156049).

**Comment by sebastienros** [comment]
Fine to remove the version, doesn't add much, the props file is a good repos for all the versions. This way we can keep the file, we don't add dependencies very often so it's low maintenance.

File: src/docs/resources/libraries/README.md
Comment (Piedone) [inline src/docs/resources/libraries/README.md:1]: The Version column headers remained (check the file in preview).

**Review by Piedone — CHANGES_REQUESTED** [review]

File: src/docs/resources/libraries/README.md
Comment (sebastienros) [reply src/docs/resources/libraries/README.md:1]: In these cases, hit '.', change the file, commit. It's even faster that commenting.
Or click Edit in the Files page, for simple changes.

Comment (Piedone) [reply src/docs/resources/libraries/README.md:1]: I was on my phone where I definitely won't go and try edit Markdown tables :).

**Review by Piedone — APPROVED** [review]

File: src/docs/resources/libraries/README.md
Comment (sebastienros) [reply src/docs/resources/libraries/README.md:1]: You need a new phone. Or an iPad with the cellular option

**Pull request merged**